aptos-move/framework/src/natives/aggregator_natives/helpers_v2.rs

```diff
@@ -1,19 +1,21 @@
 // Copyright © Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
 
-use super::helpers_v1::{get_aggregator_field, set_aggregator_field};
-use aptos_aggregator::{
-    resolver::DelayedFieldResolver,
-    types::{DelayedFieldID, SnapshotValue},
-    utils::{from_utf8_bytes, u128_to_u64},
-};
+use super::helpers_v1::set_aggregator_field;
+use aptos_aggregator::{resolver::DelayedFieldResolver, types::DelayedFieldID};
+use aptos_types::aggregator::{from_utf8_bytes, u128_to_u64};
 use move_binary_format::errors::PartialVMResult;
 use move_vm_types::values::{StructRef, Value};
 
 /// Indices of `value` and `limit` fields in the `Aggregator` Move
 /// struct.
-const VALUE_FIELD_INDEX: usize = 0;
-const LIMIT_FIELD_INDEX: usize = 1;
+pub(crate) const AGG_VALUE_FIELD_INDEX: usize = 0;
+pub(crate) const AGG_MAX_VALUE_FIELD_INDEX: usize = 1;
+
+pub(crate) const AGG_SNAPSHOT_VALUE_FIELD_INDEX: usize = 0;
+
+pub(crate) const DERIVED_STRING_VALUE_FIELD_INDEX: usize = 0;
+// pub (crate) const DERIVED_STRING_PADDING_FIELD_INDEX: usize = 1;
```

Comment:
do we need this?

Reply:
after George's change we will not need any of this indices, if I understand correctly

but here I wanted to specify the index, if we do end up needing it.